src/Native/Runtime/StackFrameIterator.cpp

```diff
@@ -614,6 +618,10 @@ void StackFrameIterator::UpdateFromExceptionDispatch(PTR_StackFrameIterator pSou
     m_RegDisplay.pR13 = thisFuncletPtrs.pR13;
     m_RegDisplay.pR14 = thisFuncletPtrs.pR14;
     m_RegDisplay.pR15 = thisFuncletPtrs.pR15;
+
+#elif defined(_TARGET_WASM_)
```

Comment:
Change this to `#elif defined(_TARGET_X86_) || defined(_TARGET_AMD64_)` and move the `PORTABILITY_ASSERT` into the `#else` block.

Reply:
Done